js/theme/viewmodel.js

```diff
@@ -114,7 +122,10 @@ function renderPosts(api_response, opts) {
  * @param {object} api_response - liveblog API response JSON.
  */
 function updateViewModel(api_response, opts) {
-  vm._items.push.apply(vm._items, api_response._items);
+  if (opts.sort === 'oldest_first')
```

Comment:
Sure, maybe we could set up some linting. The only problem being that the current style adopted is quite different from the sourcefabric one